app/components/govuk_component/header_component.rb

```diff
@@ -44,22 +44,22 @@ def initialize(classes: [],
 private
 
   def default_attributes
-    { class: %w(govuk-header) }
+    { class: ["#{ brand }-header"] }
```

Comment:
I haven't seen the spacing within an interpolated string style before. I can see how it improves readability, especially on bigger blocks of text. We now have a mix of with and without though.